.github/workflows/tests.yaml

```diff
@@ -162,6 +162,7 @@ jobs:
           CHROMEDRIVER_PATH: "${{ (matrix.browser == 'chrome' && !matrix.selenium-server) && '/usr/local/share/chromedriver-linux64/chromedriver' || '' }}"
           GECKODRIVER_PATH: "${{ (matrix.browser == 'firefox' && !matrix.selenium-server) && '/usr/local/share/gecko_driver/geckodriver' || '' }}"
           DISABLE_W3C_PROTOCOL: "${{ matrix.w3c && '0' || '1' }}"
+          DISABLE_HEADLESS: "0"
```

Comment:
```suggestion
```

As this is default behavior, I would suggest not adding this env variable for the CI run.

Reply:
I thought it would be clearer / more transparent to have it - devs would understand immediately how the pipeline behaves just by reading the file.